### PR TITLE
Eval runs

### PR DESCRIPTION
**Pull request by harry-humanloop**
Update eval utils to create run and pass run_id in log call.

Ignore the first commit by fernbot when reviewing

### CODE REVIEW

File: src/humanloop/requests/version_specification.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This file was auto-generated by Fern from our API Definition.
+
+import typing_extensions
+import typing_extensions
```
Comment (peadaroh) [inline src/humanloop/requests/version_specification.py:4]: Check where this is used

**Review by peadaroh — APPROVED** [review]
LGTM - just check the use of that legacy stuff

File: src/humanloop/requests/version_specification.py (hunk 1 above)
Comment (harry-humanloop) [reply src/humanloop/requests/version_specification.py:4]: no longer in SDK now with updated openapi spec

**Comment by harry-humanloop** [comment]
pushed (accidentally) directly to master:
https://github.com/humanloop/humanloop-python/commit/46c6504a706d0bba1097c888cb56bfc96a7df556

**Pull request closed without merging**